packages/vips/src/types.ts

```diff
@@ -0,0 +1,171 @@
+export type ItemId = string;
+
+// Same type as in @mextp/upload-media
+// TODO: Move to shared package?
+export type ImageSizeCrop = {
+	width: number;
+	height: number;
+	crop?:
+		| boolean
+		| [ 'left' | 'center' | 'right', 'top' | 'center' | 'bottom' ];
+};
+
+/**
+ * none: Do nothing. Same as low.
+ * centre: Just take the centre.
+ * entropy: Use an entropy measure
+ * attention: Look for features likely to draw human attention.
+ * low: Position the crop towards the low coordinate. Same as none.
+ * high: Position the crop towards the high coordinate.
+ * all: Everything is interesting.
+ */
+type Interesting =
+	| 'none'
+	| 'centre'
+	| 'entropy'
+	| 'attention'
+	| 'low'
+	| 'high'
+	| 'all';
```

Comment:
It jumped out at me that one place uses `center` and another `centre`. I think `centre` comes from lib vips interface. Could the package use one spelling and adjust internally?

Reply:
Yeah vips uses different spelling. But the `Interesting` type with `centre` here is not exported, it's all used internally. Used here:

https://github.com/WordPress/gutenberg/blob/1e90ef282524a82c5daa537fe1315415d243bc51/packages/vips/src/index.ts#L264